src/components/templates/__tests__/UserProfileForm.test.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Router } from "react-router";
+import { createMemoryHistory } from "history";
+import { Provider } from "react-redux";
+import { render, cleanup } from "@testing-library/react";
+import configureStore from "redux-mock-store";
+import "@testing-library/jest-dom/extend-expect";
+import UserProfileForm from "../UserProfileForm";
+import { initialState } from "../../../utils/mockData";
+
+const history = createMemoryHistory();
+
+afterEach(cleanup);
+
+let jestFeatures;
+let mockStore;
+let store;
+
+beforeEach(() => {
+  mockStore = configureStore();
+  // let wrapper;
+  store = mockStore(initialState);
+  jestFeatures = render(
+    <Router history={history}>
+      <Provider store={store}>
+        <UserProfileForm initialState={initialState} />
+      </Provider>
+    </Router>
+  );
+});
+
+describe("Shows all the text nodes on HackathonForm.js that are contained on the making an event form", () => {
+  // it("should be displaying the label text node for the Name", () => {
+  //   expect(jestFeatures.getByText("Full Name")).toBeInTheDocument();
+  // });
+  it("should be displaying the label text node for the Name", () => {
+    expect(jestFeatures).toMatchSnapshot();
```

Comment:
the description on it it() is not accurate.This is not a test to check for a label text node for the Name.
Please give it a right description.

Reply:
Also, add an extra test to check for an element of the UI.
Not just  a snaphshot test .